src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -376,4 +383,23 @@ public WebElement getModerateResponseButtonInQuestionView(int qnNo, int response
                              .findElement(By.xpath("table/tbody/tr[" + responseNo + "]/td[6]/form"));
     }
 
+    public void clickInstructorCollapseButton() {
```

Comment:
The method name and the target web element name is not in sync. One of them needs fix.